### DISCUSSION TITLE
quire epub doesn't work

### DISCUSSION THREAD

**Discussion by Mapachosa** [thegetty/quire#339] [Get Help]
Hi, 

I have a trouble with the comand `quire epub`. When I want to create the epub of my proyect in the terminal, throws me the following statement: 

Air-de-Mapache:~ mapachetorres$ cd escritos8
Air-de-Mapache:escritos8 mapachetorres$ quire epub
⠋ Building EPUBBrowserslist: caniuse-lite is outdated. Please run next command `npm update`
⠦ Building EPUBpandoc: : openBinaryFile: does not exist (No such file or directory)
✖ Error: Command failed: pandoc --from=html-native_divs+native_spans --to=epub html/epub.xhtml --output=static/downloads/output.epub --epub-metadata=html/dc.xml --epub-cover-image= --template=html/template.xhtml --css=site/css/epub.css --standalone
pandoc: : openBinaryFile: does not exist (No such file or directory)

I already installed Pandoc but I don't know what what could be happening. I appreciate that  if you can help me. Thank you

**Comment by Mapachosa** [comment]
> Hi,
> 
> I have a trouble with the comand `quire epub`. When I want to create the epub of my proyect in the terminal, throws me the following statement:
> 
> Air-de-Mapache:~ mapachetorres$ cd escritos8 Air-de-Mapache:escritos8 mapachetorres$ quire epub ⠋ Building EPUBBrowserslist: caniuse-lite is outdated. Please run next command `npm update` ⠦ Building EPUBpandoc: : openBinaryFile: does not exist (No such file or directory) ✖ Error: Command failed: pandoc --from=html-native_divs+native_spans --to=epub html/epub.xhtml --output=static/downloads/output.epub --epub-metadata=html/dc.xml --epub-cover-image= --template=html/template.xhtml --css=site/css/epub.css --standalone pandoc: : openBinaryFile: does not exist (No such file or directory)
> 
> I already installed Pandoc but I don't know what what could be happening. I appreciate that if you can help me. Thank you

**Reply by Erin-Cecele** [reply]
Hi @Mapachosa. Apologies for the delay. We are working on an answer for you and will be in touch soon!

**Reply by geealbers** [reply]
@Mapachosa I think we have a fix just pushed in the latest version of the Quire CLI (v0.20.4). Can you install it with the following command, and then let us know if you're able to create your EPUB?

```
npm install -g @thegetty/quire-cli
```

**Reply by Erin-Cecele** [reply]
Hi @Mapachosa. Let us know if this worked for you!
